SalesforceSDK/Salesforce.SDK.SmartSync/Manager/SyncManager.cs

```diff
@@ -50,27 +47,32 @@ public class SyncManager
         public const string LocallyUpdated = "__locally_updated__";
         public const string LocallyDeleted = "__locally_deleted__";
 
-        private static volatile Dictionary<string, SyncManager> _instances;
+        private static volatile Dictionary<string, SyncManager> _instances = new Dictionary<string, SyncManager>();
```

Comment:
This is set in GetInstance; there are some other pre-setup things that occur if this is not yet set.